docs/disaster-recovery.md

```diff
@@ -0,0 +1,403 @@
+  - apiGroups: ["replication.storage.openshift.io"]
+    resources: ["volumereplications/status"]
+    verbs: ["get", "patch", "update"]
+  - apiGroups: ["replication.storage.openshift.io"]
+    resources: ["volumereplicationclasses/status"]
+    verbs: ["get"]
+```
+
+### Deploy the Volume Replication Sidecar
+
+To deploy `volume-replication` sidecar container in `csi-rbdplugin-provisioner`
+ pod, add the following yaml to
+ [csi-rbdplugin-provisioner deployment](https://github.com/ceph/ceph-csi/blob/release-v3.3/deploy/rbd/kubernetes/csi-rbdplugin-provisioner.yaml).
+
+```yaml
+        - name: volume-replication
+          image: quay.io/csiaddons/volumereplication-operator:v0.1.0
+          args :
+            - "--metrics-bind-address=0"
+            - "--leader-election-namespace={{ .Namespace }}"
+            - "--driver-name=rbd.csi.ceph.com"
+            - "--csi-address=$(ADDRESS)"
+            - "--rpc-timeout=150s"
+            - "--health-probe-bind-address=:9998"
+            - "--leader-elect=true"
+          env:
+            - name: ADDRESS
+              value: unix:///csi/csi-provisioner.sock
+          imagePullPolicy: "IfNotPresent"
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+```
+
+## VolumeReplicationClass and VolumeReplication
+
+### VolumeReplicationClass
+
+*VolumeReplicationClass* is a cluster scoped resource that contains
+ driver related configuration parameters. It holds the storage admin
+ information required for the volume replication operator.
+
+### VolumeReplication
+
+*VolumeReplication* is a namespaced resource that contains references
+ to storage object to be replicated and VolumeReplicationClass
+ corresponding to the driver providing replication.
+
+>:bulb: For more information, please refer to the
+> [volume-replication-operator](https://github.com/csi-addons/volume-replication-operator).
+
+Let's say we have a *PVC* (rbd-pvc) in BOUND state; created using
+ *StorageClass* with `Retain` reclaimPolicy.
+
+```bash
+kubectl get pvc --context=cluster-1
+
+ NAME      STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS      AGE
+ rbd-pvc   Bound    pvc-65dc0aac-5e15-4474-90f4-7a3532c621ec   1Gi        RWO            csi-rbd-sc   44s
+ ```
+
+* Create Volume Replication Class on cluster-1
+
+    ```yaml
+    $cat <<EOF | kubectl --context=cluster1 apply -f -
+    apiVersion: replication.storage.openshift.io/v1alpha1
+    kind: VolumeReplicationClass
+    metadata:
+      name: rbd-volumereplicationclass
+    spec:
+      provisioner: rbd.csi.ceph.com
+      parameters:
+        mirroringMode: snapshot
+        schedulingInterval: "12m"
+        schedulingStartTime: "16:18:43"
+        replication.storage.openshift.io/replication-secret-name: csi-rbd-secret
+        replication.storage.openshift.io/replication-secret-namespace: default
+    EOF
+    ```
+
+>:bulb: **Note:** The `schedulingInterval` can be specified in formats of
```

Comment:
The `schedulingInterval` can be specified in days, hours, or minutes using the d, h, m suffix respectively. The optional `schedulingStartTime`  can be specified using the ISO 8601 time format

Reply:
Thanks for the suggestion. Added it too.